### ISSUE TITLE
Latest version bug

### ISSUE THREAD

**Issue by girlofparadise** [w2016561536/android_virtual_cam#77]
After updating to the latest version, work profile apps can't read files inside work space storage. Instead it shows toast to put files in main directory. But the apps doesn't use the files stored in main directory.

**Comment by w2016561536** [comment]
![Screenshot_20220206-232321_Island.jpg](https://user-images.githubusercontent.com/40821031/152687975-8384b685-f7df-4758-ad50-32cbdd11e022.jpg)

Sorry, I have some problems with creating the work account(with root). It will take some time to solve.

Could you tell me whether the bug exists on the older version?

**Comment by girlofparadise** [comment]
No, I'm now using 4.0 version. Its working perfectly. Try using shelter app from fdroid for work profile creation. That's what I use.

**Comment by w2016561536** [comment]
I have found the bug
It caused by replacement, and I didn't check
Thank you for your reporting

**Comment by girlofparadise** [comment]
When will you fix that bug sir?
